app/views/pieces/show.html.erb

```diff
@@ -1,32 +1 @@
-<h1> WELCOME TO YOUR GAME MWAHAHAHAH </h1>

```

Comment:
If we won't be using the pieces show page in the future, then it may make more sense to remove it entirely from routes and views